unscented_kalman_filter/cmake-build-debug/CMakeCache.txt

```diff
@@ -0,0 +1,383 @@
+# This is the CMakeCache file.
```

Comment:
As shuwei mentioned delete these files. If you are struggling with this you can add cmake-build-debug folder to .gitignore file